backend/justfile

```diff
@@ -11,8 +11,8 @@ install:
 lint:
 	isort . && black . && mypy fief/ && tryceratops fief/
 
-test $UNIT_TESTS="1": start-db
-  pytest --cov fief/ --cov-report=term-missing
+test args="--cov fief/ --cov-report=term-missing" $UNIT_TESTS="1": start-db
```

Comment:
Sorry if this is bad form. I wanted a way to be able to run something like `just test --lf`, this seemed to work? Happy to revert?

Reply:
I've applied this same idea to the new hatch stuff. Again, I hope that's OK?

Reply:
Yes, good!